### PR TITLE
Remove unused `hcb_codes/cards` partials

### PR DESCRIPTION
**Pull request by devramsean0**
## Summary of the problem

This removes some seeming unused partials under `hcb_code/cards`
See #5216 

## Describe your changes

I did a search for the partial inside VSCode, After it didn't turn anything up I deleted them

n/a

closes #5216

### CODE REVIEW

File: app/helpers/invoices_helper.rb [hunk 1]
```diff
@@ -158,13 +158,13 @@ def invoice_payout_datetime(invoice = @invoice)
...
-    title = "Funds available since"
+    title = "Funds available since "
     date = @hcb_code.canonical_transactions.pluck(:date).max
   elsif invoice.payout_creation_queued_at && invoice.payout.nil?
-    title = "Transfer scheduled"
+    title = "Transfer scheduled "
     date = invoice.payout_creation_queued_for
   elsif invoice.payout_creation_queued_at && invoice.payout.present?
-    title = "Funds should be available"
+    title = "Funds should be available "
```
Comment (sampoder) [inline app/helpers/invoices_helper.rb]: These changes are probably unrelated

Comment (garyhtou) [reply app/helpers/invoices_helper.rb]: looks like they came from https://github.com/hackclub/hcb/pull/5273

Comment (garyhtou) [reply app/helpers/invoices_helper.rb]: I've discarded these unrelated changes from this PR
[`5c1298c` (#5274)](https://github.com/hackclub/hcb/pull/5274/commits/5c1298c47c78aaa0d96ade248871fcc23fc61e03)

**Comment by devramsean0** [comment]
Thanks lol, I swear my state was clean before I switched branches

**Review by sarthaktexas — APPROVED** [review]

**Review by garyhtou — APPROVED** [review]
great!

**Pull request merged**